src/games/rcll/orders.clp

```diff
@@ -60,36 +60,10 @@
 )
 
 
-(defrule order-recv-SetOrderDeliveredByColor
```

Comment:
Can we just keep the OrderDeliveredByID menu/message/option? I think that would be sufficient for any fallback scenarios and also for testing.

Reply:
Yes, as I mentioned in my general comment I agree, it removes a lot of code and we still have a fall-back. The current refbox for the 2018 simulation should then be tagged so that the game can still be played if necessary.